build.sbt

```diff
@@ -6,15 +7,15 @@ lazy val root = (project in file(".")).
     name                := "play-json-xml",
     organization        := "org.micchon",
     scalaVersion        := Scala212,
-    crossScalaVersions  := Scala212 :: Scala211 :: Nil,
+    crossScalaVersions  := Scala213 :: Scala212 :: Scala211 :: Nil,
```

Comment:
Could you drop 2.11? I think 2.11 version is no longer needed.

Reply:
Done in 3d7b555

Reply:
I did not change library version, but I guess it should be incremented. Otherwise someone using Scala 2.11 and relying on 0.4.3 to have it, would be in for a surprise.

Reply:
> but I guess it should be incremented.

Absolutely yes.  If this commit is merged, I will do that.